tools/gvm-lsc-deb-creator

```diff
@@ -161,7 +161,7 @@ COPYRIGHT_FILE="${DOC_DATA_DIR}/copyright"
 
 # Create data archive
 cd "${DATA_DIR}"
-tar -C "${DATA_DIR}" -acf "../data.tar.xz" "${HOME_SUBDIR}" "${DOC_SUBDIR}"
+tar -C "${DATA_DIR}" -acf "../data.tar.gz" "${HOME_SUBDIR}" "${DOC_SUBDIR}"
```

Comment:
Are you even sure that this is actually compressed as an `xz`?
Afaik you need to set the `-J` option for that compression format or pipe the `tar` archive to `xy` or `gzip`?